KarhooUISDK/Builders/CheckoutScreenBuilder.swift

```diff
@@ -11,22 +11,24 @@ import KarhooSDK
 
 public protocol CheckoutScreenBuilder {
     func buildCheckoutScreen(
-        quote: Quote,
+        navigationController: UINavigationController?,
+        quote: KarhooSDK.Quote,
         journeyDetails: JourneyDetails,
-        bookingMetadata: [String: Any]?,
+        bookingMetadata: [String : Any]?,
         callback: @escaping ScreenResultCallback<KarhooCheckoutResult>
-    ) -> Screen
+    ) -> KarhooUISDKSceneCoordinator
 }
 
 public extension CheckoutScreenBuilder {
     func buildCheckoutScreen(
-        quote: Quote,
+        navigationController: UINavigationController?,
+        quote: KarhooSDK.Quote,
```

Comment:
Why not just `import KarhooSDK` instead of adding it everywhere we use something from it?

Reply:
Great catch. It was an old, temporary, implementation.